File: Back/.Net/BetFriends.Domain/AnswerBets/IAnswerBetRepository.cs
using BetFriends.Domain.Bets;

namespace BetFriends.Domain.AnswerBets;

public interface IAnswerBetRepository
{
    Task<IEnumerable<AnswerBet>> GetAnswersAsync(BetId betId);
    Task SaveAsync(AnswerBet answerBet);
}

File: Back/NodeJS/modules/bets/src/domain/answerBets/IAnswerBetRepository.ts
import { BetId } from "../bets/BetId";
import { AnswerBet } from "./AnswerBet";

export interface IAnswerBetRepository {
    GetAnswersForBet(BetId: BetId): Promise<AnswerBet[]>;
    Save(answerBet: AnswerBet): Promise<void>;

}